jujugui/static/gui/src/app/app.js

```diff
@@ -2649,7 +2655,7 @@ YUI.add('juju-gui', function(Y) {
        @type {Boolean}
        @default false
        */
-      interactiveLogin: {value: false},
+      interactiveLogin: { value: true },
```

Comment:
Thanks! Well, except for the space inside braces. But that's how life goes, you'll keep adding them, I'll keep removing them...